dsc-website-client/src/App.js

```diff
@@ -21,6 +21,7 @@ function App() {
           <Route exact path="/contact" component={Contact} />
           <Route exact path="/faq" component={Faq} />
           <Route exact path="/pastevents" component={PastEvents} />
+          <Route exact path="/timeline" component={CustomizedTimeline} />
```

Comment:
Could you change this name to /pastEvents or something along those lines?

Reply:
sure!